pom.xml

```diff
@@ -70,7 +70,7 @@ limitations under the License. -->
     <!-- Rerun failed unit tests up to 3 times -->
     <surefire.rerunFailingTestsCount>3</surefire.rerunFailingTestsCount>
 
-    <failsafe.forkCount>4</failsafe.forkCount>
+    <failsafe.forkCount>1</failsafe.forkCount>
```

Comment:
So we always run ITs sequentially?

Reply:
Got it

Reply:
Maybe try to run at least UTs in parallel if it help to reduce time to run?

Reply:
These UTs take only 3 minutes and performed in parallel with ITs. I can return 4 threads here, but is it worth it? Not sure if UTs logs are mixed, checked this only for ITs.

Reply:
Logs should be mixed for UTs as well, so if it takes only 3 minutes, lets keep it 1 thread only

Reply:
I set the thread count to 4, and it takes 2 minutes instead of 3 now.